ur_controllers/src/force_torque_sensor_controller.cpp

```diff
@@ -0,0 +1,180 @@
+//
+// Created by livanov93 on 12/12/20.
+//
+
+#include "ur_controllers/force_torque_sensor_controller.h"
+
+namespace ur_controllers
+{
+ForceTorqueStateController::ForceTorqueStateController() : controller_interface::ControllerInterface()
+{
+}
+
+controller_interface::return_type ForceTorqueStateController::init(const std::string& controller_name)
```

Comment:
nit pick (to get it easier into `ros2_controllers`)

```suggestion
controller_interface::return_type ForceTorqueStateController::init(const std::string & controller_name)
```